src/FacebookLocales.js

```diff
@@ -60,6 +60,14 @@ const localesToNonStandardFacebookLocales = _(facebookVirtualLocales)
 	.value()
 
 export const bestFacebookLocaleFor = locale => {
+	// Normalize locale input format
+	if (locale.includes('-')) {

```

Comment:
The `if (locale.includes('-')) {` check is moot for this as `replace` is a noop if there's no `-` in the string.

Reply:
I treat this just about kind of style, I know the results is totally the same :)